src/Joystick/Joystick.cc

```diff
@@ -214,7 +238,7 @@ float Joystick::_adjustRange(int value, Calibration_t calibration)
                             << calibration.min
                             << calibration.max
                             << calibration.center
-                            << calibration.center
+                            << calibration.deadband
```

Comment:
Remove center? Shouldn't this be like the others with an add of deadband? Also not sure if there is text update needed here.